docs/BR.md

```diff
@@ -996,25 +1026,92 @@ Databases maintained by the CA, its owner, or its affiliated companies do not qu
+
+If any of the above considerations are performed by a Delegated Third Party, the CA MAY obtain reasonable evidence from the Delegated Third Party to ascertain assurance that one or more of the above considerations are followed. As an exception to Section 1.3.2, Delegated Third Parties are not required to be within the audit scope described in Section 8 of these Requirements to satisfy the above considerations.
+
+Phased Implementation Timeline:
+- *Effective September 15, 2024*, the CA SHOULD implement Multi-Perspective Issuance Corroboration using at least two (2) remote Network Perspectives. 
+- *Effective March 15, 2025*, the CA MUST implement Multi-Perspective Issuance Corroboration using at least two (2) remote Network Perspectives. The CA MAY proceed with certificate issuance if the number of remote Network Perspectives that do not corroborate the determinations made by the Primary Network Perspective ("non-corroborations") is greater than allowed in the Quorum Requirements table.  
+- *Effective September 15, 2025*, the CA MUST implement Multi-Perspective Issuance Corroboration using at least two (2) remote Network Perspectives. The CA MUST NOT proceed with certificate issuance if the number of non-corroborations is greater than allowed in the Quorum Requirements table.
+- *Effective March 15, 2026*, the CA MUST implement Multi-Perspective Issuance Corroboration using at least three (3) remote Network Perspectives. The CA MUST NOT proceed with certificate issuance if the number of non-corroborations is greater than allowed in the Quorum Requirements table and if the remote Network Perspectives that do corroborate the determinations made by the Primary Network Perspective do not fall within the service regions of at least two (2) distinct Regional Internet Registries.  
+- *Effective June 15, 2026*, the CA MUST implement Multi-Perspective Issuance Corroboration using at least four (4) remote Network Perspectives. The CA MUST NOT proceed with certificate issuance if the number of non-corroborations is greater than allowed in the Quorum Requirements table and if the remote Network Perspectives that do corroborate the determinations made by the Primary Network Perspective do not fall within the service regions of at least two (2) distinct Regional Internet Registries. 
+- *Effective December 15, 2026*, the CA MUST implement Multi-Perspective Issuance Corroboration using at least five (5) remote Network Perspectives. The CA MUST NOT proceed with certificate issuance if the number of non-corroborations is greater than allowed in the Quorum Requirements table and if the remote Network Perspectives that do corroborate the determinations made by the Primary Network Perspective do not fall within the service regions of at least two (2) distinct Regional Internet Registries.
 
```

Comment:
Would it be relevant to acknowledge MPIC was already deployed by some CAs in advance of these dates and edits?  And is this document to be silent about how MPIC could be used prior to September 15?

Reply:
@miqrogroove - thanks for your comments.

> Would it be relevant to acknowledge MPIC was already deployed by some CAs in advance of these dates and edits?

While acknowledging early adoption of MPIC is commendable, it's not directly relevant to the requirements language of the proposed BR update.  Including such information would be transient and require removal in later updates. We do appreciate those CAs who have adopted or plan to adopt MPIC in advance of the ballot's potential acceptance.

> And is this document to be silent about how MPIC could be used prior to September 15?

You're correct. The update is deliberately silent on MPIC usage prior to September 15, 2025. This approach ensures that early adopters (1) are not inadvertently penalized for going beyond the *minimum* requirements described in the BRs, and  (2) share the same opportunity to comply with the proposed requirements once they are intended to take effect.